.github/workflows/test-changed-firestore.yml

```diff
@@ -35,23 +35,27 @@ jobs:
           yarn
       - name: build
         id: build
-        # TODO(wuandy): Separate yarn and egrep into steps, so build failure
-        # is captured by github actions.
-        run: yarn build:changed firestore | egrep "Skipping all"
+        run: |
+          set -o pipefail
+          yarn build:changed firestore | tee ${{ runner.temp }}/yarn.log.txt
+        continue-on-error: false
```

Comment:
Is `continue-on-error: false` required? Isn't false the default?

Reply:
No required, but I feel safer to keep it there.